### PR TITLE
Setup hello-world backend API

### PR DESCRIPTION
**Pull request by sharon-odhiambo**
In this PR I used a created a rails backend API to be connected with the react front-end for displaying random messages..

**Below are the highlights**

- Use Ruby on Rails to create new app
- Create a Greetings backend API endpoint
- Use Postgresql for setting up and storing the database
- Implement the correct Gitflow
- Ensure no linter errors

**Cheers and Happy Reviewing!**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  namespace :api do
+    resources :greetings, only: [:index]
+  end
```
Comment (FrederickMih) [inline config/routes.rb:4]: - [ ] Based on the requirements of this project, the two apps should work together. Right now, the API endpoint for returning random greetings returns **null** hence could not connect properly with the client side. Kindly fix the issue


Screenshot from 2022-12-01 19-25-38

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @sharon-odhiambo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights

- Correct routes have been added :heavy_check_mark: 
- PR is descriptive :heavy_check_mark: 
- Linters are passing :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sharon-odhiambo** [comment]
hello @FrederickMih , thank you for the review.

**Review by descholar-ceo — APPROVED** [review]
Hi @sharon-odhiambo 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented the requested changes ✔️ 
👍🏽 The PR is following the correct  git flow ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sharon-odhiambo** [comment]
Hello @descholar-ceo,thank you for the review.

**Pull request merged**